com.unity.render-pipelines.high-definition/Runtime/Lighting/Light/HDAdditionalLightData.Types.cs

```diff
@@ -96,6 +96,7 @@ public enum ShadowUpdateMode
     }
 
     /// <summary>Light Layers.</summary>
+    [Flags]
```

Comment:
The same flag might also need be added on the DecalLayerEnum in DecalSystem.cs.

Reply:
Good catch! this is added to this PR